### PR TITLE
Adding reference how to do configmap changes

### PR DESCRIPTION
**Pull request by ProbStub**
Hello, 

In reference to https://stackoverflow.com/questions/54571185/how-to-patch-a-configmap-in-kubernetes adding a note on how to modify configmaps in a deployment.

Modifying config maps is an important task and should be found easily from the basic configmap introductions.

### CODE REVIEW

File: content/en/docs/concepts/configuration/configmap.md [hunk 1]
```diff
@@ -279,5 +279,6 @@ to the deleted ConfigMap, it is recommended to recreate these pods.
 
 * Read about [Secrets](/docs/concepts/configuration/secret/).
 * Read [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+* Read about [changing a ConfigMap (or any other Kubernetes object)] (/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)
```
Comment (jihoon-seo) [inline content/en/docs/concepts/configuration/configmap.md:282]: To make the link work:

```suggestion
* Read about [changing a ConfigMap (or any other Kubernetes object)](/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)
```

**Review by jihoon-seo — COMMENTED** [review]
Preview: https://deploy-preview-33418--kubernetes-io-main-staging.netlify.app/docs/concepts/configuration/configmap/#what-s-next

@ProbStub PTAL!

**Review by divya-mohan0209 — COMMENTED** [review]
https://deploy-preview-33418--kubernetes-io-main-staging.netlify.app/docs/concepts/configuration/configmap/ - This doesn't look fine.

https://kubernetes.io/docs/concepts/configuration/configmap/ - The existing link works fine.

Closing the PR since there is no need for an amendment.

**Comment by divya-mohan0209** [comment]
Closing the PR per [this comment](https://github.com/kubernetes/website/pull/33418#pullrequestreview-961357517)

File: content/en/docs/concepts/configuration/configmap.md (hunk 1 above)
Comment (ProbStub) [reply content/en/docs/concepts/configuration/configmap.md:282]: Wonderful. Thank you and apologies. Will try with a new PR since this is close.

Comment (divya-mohan0209) [reply content/en/docs/concepts/configuration/configmap.md:282]: @ProbStub : I do not see the requirement for a change in the first place. The existing link and content seem okay to me. I don't think you need to make amendments with another PR.

Comment (ProbStub) [reply content/en/docs/concepts/configuration/configmap.md:282]: Your call. I spent 2h to find a way how to update existing objects. If there is information on how to create and use ConfigMap perhaps users should also know how to change ConfigMaps.

Comment (divya-mohan0209) [reply content/en/docs/concepts/configuration/configmap.md:282]: @ProbStub To clarify what I meant, if you're adding a new link you don't need to open a new PR. You can reopen this one.

Comment (ProbStub) [reply content/en/docs/concepts/configuration/configmap.md:282]: @divya-mohan0209 I see. Thank you for reopening :)

Comment (divya-mohan0209) [reply content/en/docs/concepts/configuration/configmap.md]: Apologies for the unintentional miscommunication.

Comment (ProbStub) [reply content/en/docs/concepts/configuration/configmap.md]: No worries. It is my first contribution here so am happy for the detailed review. Thanks for all the feedback!

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Pull request merged**